python/ray/worker.py

```diff
@@ -1729,7 +1729,8 @@ def wait(object_ids, num_returns=1, timeout=None, worker=global_worker):
     return ready_ids, remaining_ids
 
 
-def wait_for_function(function_id, driver_id, timeout=5, worker=global_worker):
+def wait_for_function(function_id, driver_id, timeout=10,
```

Comment:
This is unrelated to the bug. However, it's a reasonable change, so we can leave it in.